server/models/polls.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const PollSchema = new Schema({
+	title: {
+		type: String,
+		required: true,
+	},
+	userId: {
+		type: Number,
+		required: true,
+	},
+	date: {
+		type: Date,
+		default: Date.now,
+	},
+	images: [
+		{
+			url: {
+				type: String,
+				required: true,
+			},
+		},
+	],
+});
+
+PollSchema.methods.addImage = function (imageLocation) {
+	this.images.push({ url: imageLocation });
+};
+
+PollSchema.methods.updateImage = function (imageLocation, id) {
+	let imageObj = this.images.find((image) => String(image._id) === id);
```

Comment:
use const instead of let here (whenever you can, use const)

Reply:
I used let instead of const in this one, because, I was not sure if I had to make a change or not, and in case I'd needed it, I will need to have it as a let.